authenticated-digital-emblem-scoping.md

```diff
@@ -2,11 +2,11 @@
 
 ## Background
 
-Under international humanitarian law, parties to armed conflict are obliged to refrain from attacking infrastructure marked with an emblem or distinctive sign, including the Red Cross, Red Crescent and Red Crystal. As conflict now involves cyberattack on digital resources, clear authenticated communication of equivalent digital emblems can facilitate compliance with international humanitarian law and protection of humanitarian resources. Extending digital emblems must be accessible through well-known, consensus mechanisms that are respected, verifiable, and accountable against misuse.
+Under international humanitarian law, parties to armed conflict are obliged to refrain from attacking physical infrastructure marked with an emblem or distinctive sign, including the Red Cross, Red Crescent and Red Crystal. As conflict now involves cyberattack on digital resources, clear authenticated communication of equivalent digital emblems for digital assets can facilitate compliance with international humanitarian law and protection of humanitarian resources. Extending digital emblems must be accessible through well-known, consensus mechanisms that are respected, verifiable, and accountable against misuse.
```

Comment:
I'm not sure IHL currently limits protections to physical infrastructure, which this could be read as. But appreciate the clarification of scope to digital assets.